src/services/infra/__tests__/DynamoDBService.spec.ts

```diff
@@ -41,7 +41,7 @@ const mockFailureLaunch: SiteLaunchMessage = {
   },
 }
 
-const tableName = "site-launch"
+const tableName = "ddb-name"
```

Comment:
We've changed this on AWS already yeah?

Reply:
the prod and staging ebs already have the updated table names. This PR only fixes the failing tests